docs/RandomForrest/user_guide/chosing_n_jobs.md

```diff
@@ -0,0 +1,3 @@
+# How to chose the n_jobs parameter of RandomForest
+When creating a random forest you can supply the parameter `n_jobs`[Link to parameter]. As stated in the documentation this controls the number of processes used when fitting and predicting the random forest. In order to allow for the creation of a custom [criteria function](../../decision_tree/examples/creatingACriteria.md) without requiring that the [GIL](https://wiki.python.org/moin/GlobalInterpreterLock) is released, we opted for multiprocessing instead of multithreading. We implemented multiprocessing using Python's [multiprocessing library](https://docs.python.org/3/library/multiprocessing.html). 
+The overhead involved in setting up each new process when doing multiprocessing is rather large. Thus there is a trade-off between the overhead introduced in setting up each new process versus the amount of work that each process gets to do. For a lower amount of trees or smaller datasets this often results in diminishing returns on the speed up gained from using more processors. Since the `n_jobs` parameter controls the amount of processors used, it is advisible to investigate the speed up gained when fitting and predicting using different values for the parameter `n_jobs`.
```

Comment:
```suggestion
The overhead involved in setting up each new process when doing multiprocessing is rather large. Thus there is a trade-off between the overhead introduced in setting up each new process versus the amount of work that each process gets to do. For a lower amount of trees or smaller datasets this often results in diminishing returns on the speed up gained from using more processors. Since the `n_jobs` parameter controls the amount of processors used, it is advisable to investigate the speed up gained when fitting and predicting using different values for the parameter `n_jobs`.
```